docs/core/versions/selection.md

```diff
@@ -25,7 +25,12 @@ The rest of this document examines those four scenarios.
 
 ## The SDK uses the latest installed version
 
-SDK commands include `dotnet new`,  , or `dotnet run`. The `dotnet` CLI must choose an SDK version for any command. The .NET Core CLI uses the latest SDK installed on the machine by default. You'll use the .NET Core SDK v2.1.301 when it's installed, even if the project you are working with targets the .NET Core Runtime 2.0. You will use the latest preview versions as well as released versions. You can take advantage of the latest SDK features and improvements while targeting earlier .NET Core runtime versions. You can target multiple runtime versions of .NET Core on different projects, using the same SDK tools for all projects.
+SDK commands include `dotnet new` and `dotnet run`. The `dotnet` CLI must choose an SDK version for every dotnet command. The .NET Core CLI uses the latest SDK installed on the machine by default, even if:
+
+* The project targets an earlier version.
+* The latest version is a preview version.
+
+Apps can take advantage of the latest SDK features and improvements while targeting earlier .NET Core runtime versions. You can target multiple runtime versions of .NET Core on different projects, using the same SDK tools for all projects.
```

Comment:
>Apps can take advantage of the latest SDK features

About what "apps" do we talk here? If about the ones that are deployed to clients, then it reads that SDK also must be deployed, because "apps can take advantage of the latest SDK features".

I agree with the [comment](https://github.com/dotnet/docs/pull/7695#discussion_r218825582) of @tdykstra: it should be "you" here:
>You can take advantage of the latest SDK features and improvements while working on the project that targets an earlier version of the .NET Core runtime.

While "you" use tools, it's the "project" that targets runtime. So, the second sentence sounds better like that:
> You can use the same SDK tools to work on multiple projects that target various .NET Core runtime versions.

Though, I'm not sure if the second sentence is necessary at all. Seems that it can be removed at all.